crypto/info.c

```diff
@@ -221,7 +221,7 @@ const char *OPENSSL_info(int t)
     case OPENSSL_INFO_CONFIG_DIR:
         return ossl_get_openssldir();
     case OPENSSL_INFO_ENGINES_DIR:
-        return ossl_get_enginesdir();
+        return NULL;
```

Comment:
NIT: I think we can just delete the case branch here, because if nothing is found the function returns NULL anyway.

Reply:
yes, but I think it improves code readability - you can immediately see that it is the intention to stub this option (and that it is just a forgotten case).